samples/tab-add-in-combined/nodejs/add-in/webpack.config.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-undef */
+
+const devCerts = require("office-addin-dev-certs");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+const urlDev = "https://localhost:3000/";
+const urlProd = "https://www.blueyonderairlines.com/"; // CHANGE THIS TO YOUR PRODUCTION DEPLOYMENT LOCATION
```

Comment:
What are we going to deploy on this URL?

Reply:
Will cover in different new PR